src/it/resources/wiremock/__files/stack-set.json

```diff
@@ -82,7 +82,7 @@
       },
       {
         "serviceName": "my-test-stackset-svc2",
-        "servicePort": 8080,
+        "servicePort": "http",
```

Comment:
Maybe add a new test instead of updating the existing

Reply:
This covers both. There is both a named and numeric port in this test. A separate test would be 100s of lines of new wiremocked calls, which I want to avoid